tests/ProxyManagerTest/Factory/LazyLoadingGhostFactoryTest.php

```diff
@@ -102,6 +102,39 @@ public function testWithOptionalFactory()
     }
 
     /**
+     * @dataProvider skipPropertiesFixture
+     */
+    public function testSkipProperties($className, $properties)
+    {
+        $propertyName = $properties[0]; 
+        $factory      = new LazyLoadingGhostFactory();
+        $ghostObject  = $factory->createProxy(
+            $className,
+            function () use ($propertyName) {
+                throw new \RunTimeException(
+                    sprintf('The property %s not need to be lazy loaded', $propertyName)
+                );
+            },
+            $properties
+        );
+
+        $reflection = new \ReflectionClass($className);
+        $property   = $reflection->getProperty($propertyName);
```

Comment:
This line is wrong: something is wrong with the test if this particular call is succeeding